external/storm-redis/src/main/java/org/apache/storm/redis/common/config/JedisClusterConfig.java

```diff
@@ -44,14 +45,16 @@
      * @param nodes list of node information for JedisCluster
      * @param timeout socket / connection timeout
      * @param maxRedirections limit of redirections - how much we'll follow MOVED or ASK
+     * @param password password, if any
      * @throws NullPointerException when you didn't set nodes
      */
-    public JedisClusterConfig(Set<InetSocketAddress> nodes, int timeout, int maxRedirections) {
+    public JedisClusterConfig(Set<InetSocketAddress> nodes, int timeout, int maxRedirections, String password) {
```

Comment:
Though we also have builder class as well, it would be better to have a separate constructor which doesn't have password as parameter, given that it was `public`.